firmwares/piezo_i2c_backpack.ino

```diff
@@ -81,18 +81,23 @@ void onReceive(int howMany) {
     Serial.println(ms);
     Serial.println("--------------------");
   #endif
-
+  Serial.println(OUTPUT);
   pinMode(pin, OUTPUT);
 
   if (command == NO_TONE) {
     noTone(pin);
   }
 
+  // delay between notes
+  int pauseBetweenNotes = ms * 1.3;
+  delay(pauseBetweenNotes);
+  noTone(pin);
```

Comment:
This should be inside the `if (command == TONE) {` body, otherwise it's unnecessarily called for `NO_TONE` commands as well